src/Microsoft.HttpRepl/ApiConnection.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.HttpRepl
     internal class ApiConnection
     {
         private readonly IPreferences _preferences;
-        private const string SwaggerSearchPaths = "swagger.json|swagger/v1/swagger.json|/swagger.json|/swagger/v1/swagger.json";
+        private const string SwaggerSearchPaths = "swagger.json|swagger/v1/swagger.json|openapi.json|/swagger.json|/swagger/v1/swagger.json|/openapi.json";
```

Comment:
If its common to use those paths without the extension, then we should probably add it. I will note that the OAS spec suggests paths _with_ the extension though, so that's what I started with here. Can always add more later.

Reply:
is `oas` a known term or is this something we are causally using and/or inventing?